app/controllers/code_schools_controller.rb

```diff
@@ -7,4 +7,16 @@ def index
     @code_schools_by_state = code_schools.by_state
     @country = Country.named('United States')
   end
+
+  def map
+    code_schools = CodeSchools.new.all
+    @lat_longs = []
+
+    code_schools.each do |hash| 
```

Comment:
It looks like this is pulling out the lat/longs for each school. I was wondering if it makes more sense to have this in the `CodeSchools` class as something like `CodeSchools.with_lat_longs` or something like that.